model/project_parser.go

```diff
@@ -436,6 +436,8 @@ type parserBVTaskUnit struct {
 	Activate *bool `yaml:"activate,omitempty" bson:"activate,omitempty"`
 	// TaskGroup is set if an inline task group is defined on the build variant config.
 	TaskGroup *parserTaskGroup `yaml:"task_group,omitempty" bson:"task_group,omitempty"`
+	// CreateCheckRun will create a check run on GitHub if set
```

Comment:
sorry for all the nits about periods :shipit:

Reply:
np! fixed the periods and added a test that reads from a yaml.